src/components/Sidebar/SidebarFooter.tsx

```diff
@@ -1,81 +0,0 @@
-import { Box, Divider, HStack, Icon, IconButton, Link } from "@chakra-ui/react"
-import { BsDiscord, BsGithub } from "react-icons/all"
-import { useSidebar } from "../../hooks/useSidebar"
-import { BodySm } from "@threshold-network/components"
-import { ExternalHref } from "../../enums"
-import { FC } from "react"
-import useChakraBreakpoint from "../../hooks/useChakraBreakpoint"
-import NavItem from "./NavItem"
-
-const FooterItem: FC<{ href: string; icon: any; text: string }> = ({
-  href,
-  icon,
-  text,
-}) => {
-  return (
-    <Box
-      as={Link}
-      href={href}
-      target="_blank"
-      _hover={{
-        textDecoration: "none",
-        color: "gray.700",
-        cursor: "pointer",
-      }}
-    >
-      <IconButton
-        variant="side-bar"
-        aria-label={text}
-        icon={<Icon as={icon} />}
-      />
-    </Box>
-  )
-}
-
-const SidebarFooter = () => {
-  const { isOpen } = useSidebar()
-  const isMobile = useChakraBreakpoint("md")
-  return (
-    <Box as="footer">
-      <Divider w="auto" marginX={isOpen ? "18px !important" : "8px"} />
-      {isMobile ? (
-        <Box p={8}>
-          <HStack justify="center">
-            <FooterItem
-              text="Github"
-              href={ExternalHref.thresholdGithub}
```

Comment:
For now, let's remove `thresholdGithub` from `ExternalHref`. We can add it later if needed. It point to the wrong repo anyway.

Reply:
Ref commit: eb2a53a2